tutorials/graphs/scatter2.C

```diff
@@ -0,0 +1,42 @@
+/// \file
+/// \ingroup tutorial_graphs
+/// \notebook
+/// Draw a scatter plot.
+///
+/// \macro_image
+/// \macro_code
+///
+/// \author Olivier Couet
```

Comment:
I am not the author ;-)

Reply:
I copy-pasted most parts of the tutorial from you ;)